vnext/docs/ProjectStructure.md

```diff
@@ -0,0 +1,100 @@
+# MSBuild Project Structure
+
+This is a description of each of the core projects, their purpose and output artifacts.<br/>
+Sample applications are not covered.
+
+## Alphabetical Index
+- [Chakra\Chakra.vcxitems](#Chakra)
+- [Desktop\React.Windows.Desktop.vcxproj](#React.Windows.Desktop)
+- **[Desktop.DLL\React.Windows.Desktop.DLL.vcxproj](#React.Windows.Desktop.DLL)**
+- [Desktop.IntegrationTests\React.Windows.Desktop.IntegrationTests.vcxproj](#React.Windows.Desktop.IntegrationTests)
+- [Desktop.UnitTests\React.Windows.Desktop.UnitTests.vcxproj](#React.Windows.Desktop.UnitTests)
+- [Folly\Folly.vcxproj](#Folly)
+- [FollyWin32\FollyWin32.vcxproj](#FollyWin32)
+- [IntegrationTestScripts\IntegrationTests.njsproj](#IntegrationTests-Node-Project)
+- [IntegrationTests\React.Windows.IntegrationTests.vcxproj](#React.Windows.IntegrationTests)
+- [ReactCommon\ReactCommon.vcxproj](#ReactCommon)
+- **[ReactUWP\ReactUWP.vcxproj](#ReactUWP)**
+- [ReactWindowsCore\ReactWindowsCore.vcxproj](#ReactWindowsCore)
+- [Shared\Shared.vcxitems](#Shared)
+- [Universal.IntegrationTests\React.Windows.Universal.IntegrationTests.vcxproj](#React.Windows.Universal.IntegrationTests)
+- [Universal.UnitTests\React.Windows.Universal.UnitTests.vcxproj](#React.Windows.Universal.UnitTests)
+
+## Common Projects
+
+### ReactWindowsCore
+*Static Library*<br/>
+Contains common functionality for both Desktop and Universal Windows variants.
+
+### Shared
+*Shared Items (no build artifact)*<br/>
+Holds sources common to both Windows variants, that require different build configuration
+(i.e. compiler flags, language standard).
+
+### ReactCommon
+*Static Library*<br/>
+React Native core, cross-platform C++ types and interfaces.<br/>
+Sources provided as part of the `react-native` Node dependency. Not part of this repository.<br/>
+See https://github.com/facebook/react-native/tree/v0.59.9/ReactCommon.
+
+### Folly
+*Static Library*<br/>
+Folly variant type system for JavaScript/C++ interoperability.<br/>
+Sources provided as part of the `react-native` Node dependency. Not part of this repository.
+
+### Chakra
+*Shared Items (no build artifact)*<br/>
+ChakraCore bridging layer. May use different compiler flags between Windows variants.
+
+### React.Windows.IntegrationTests
+*Static Library*<br/>
+Common framework for running out of process and/or full React instance testing.
+
+### IntegrationTests (Node Project)
+*MSBuild Node project. For reading/editing purposes only (no build artifact)*<br/>
+Set of JavaScript component tests for [RNTester](https://github.com/facebook/react-native/tree/v0.59.9/RNTester).
+Sources provided as part of the `react-native` Node dependency. Not part of this repository.<br/>
+See https://github.com/facebook/react-native/tree/v0.59.9/IntegrationTests.
+
+## Windows Desktop Projects
+
+### React.Windows.Desktop
```

Comment:
>Desktop [](start = 18, length = 7)

Would it make sense to describe the name parts (e.g. "Desktop" = "building for Win32 apps", "Universal" = "building for UWP apps" ...)? This might enable contributors to create conforming project/folder names. Perhaps this could be part of a broader coding style/project conventions doc.

Reply:
Makes sense.

Reply:
On second thought, don't think this clarification belongs in this document.
May update the main README file.